devtools/aesh/src/main/java/io/quarkus/cli/commands/ListExtensionsCommand.java

```diff
@@ -23,10 +23,10 @@
     @Option(shortName = 'h', hasValue = false)
     private boolean help;
 
-    @Option(shortName = 'a', hasValue = false, description = "Display name, group-id, artifact-id and version (default behaviour)")
+    @Option(shortName = 'a', hasValue = false, description = "Display all extensions or just the installable.")
     private boolean all = false;
 
-    @Option(shortName = 'f', hasValue = true, description = "Display in simplified format, only display the extension names or full format")
+    @Option(shortName = 'f', hasValue = true, description = "Choose between 3 output formats: name, simple, full.")
```

Comment:
What about `Select the output format among 'name' (display the name only), 'simple' (display ...) and 'full' (default).`

I also wondering if `simple` should not be replaced by `concise`.